.docker/entrypoint/spec/helpers/spec_jekyll_build.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Jekyll
+  module PlantUml
+    module Specs
+      module Helpers
+        class SpecJekyllBuild
```

Comment:
I'd like to see a comment on what this is supposed to do. Unles this syntax makes sense for an experienced Ruby developer.